Source package : python-pylatex
File name       : PyLaTeX-1.4.2.tar.gz
Lookaside URL   : https://src.fedoraproject.org/repo/pkgs/python-pylatex/PyLaTeX-1.4.2.tar.gz/sha512/dcbe58de0ab715d6fbb571a7ad3981c875eacd08d06244b8715fa240df6f8f072417ae076a5caee204e000bf7eb55097b9ba49eddb5ce666470e494c15aa88ed/PyLaTeX-1.4.2.tar.gz
Source URL      : https://github.com/JelteF/PyLaTeX/archive/v1.4.2/PyLaTeX-1.4.2.tar.gz

source file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/source-PyLaTeX-1.4.2.tar.gz: gzip compressed data, from Unix, original size modulo 2^32 727040
lookaside file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/lookaside-PyLaTeX-1.4.2.tar.gz: gzip compressed data, from Unix, original size modulo 2^32 4014958588 gzip compressed data, unknown method, ASCII, has CRC, extra field, encrypted, last modified: Thu Sep 21 21:43:35 2034, original size modulo 2^32 4014958588


diff -r source-PyLaTeX-1.4.2.tar.gz-extracted/PyLaTeX-1.4.2/pylatex/_version.py lookaside-PyLaTeX-1.4.2.tar.gz-extracted/PyLaTeX-1.4.2/pylatex/_version.py
28c28
<     git_refnames = " (tag: v1.4.2)"
---
>     git_refnames = " (HEAD -> master, tag: v1.4.2, refs/pull/191/head)"
